code/modules/client/preference/character.dm

```diff
@@ -2086,3 +2094,20 @@
 
 	from_db = FALSE
 	return TRUE
+
+/datum/character_save/proc/init_custom_emote_text(overrides, raw)
+	if(raw)
+		try
+			overrides = json_decode(raw)
```

Comment:
This probably can go now, since we json decode as part of a sanitize_json check in `character.dm`

Reply:
yeeted